tsconfig.json

```diff
@@ -1,15 +1,9 @@
 {
   "compilerOptions": {
-    "target": "ES2017",
-    "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true,
-    "sourceMap": false,
-    "moduleResolution": "node",
-    "noImplicitAny": true,
-    "downlevelIteration": true,
-    "noImplicitThis": true,
+    "target": "ESNext",
+    "module": "ESNext",
+    "moduleResolution": "Node",
```

Comment:
We'll have to see if we can change this to `Node16` or `NodeNext`, but since some of our dependencies ship with broken types we cannot do so now.